### PR TITLE
Style forms, pages, and links with bootstrap

### PR DESCRIPTION
**Pull request by DariaSova**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
```
Comment (Sinetheta) [inline app/assets/stylesheets/application.scss:2]: You might want to also add `@import "bootstrap/theme";` then you can delete a bunch of your styles.

File: app/assets/stylesheets/components/_buttons.scss [hunk 2]
```diff
@@ -8,3 +8,7 @@ input[type="submit"],
   border: 0;
   padding: 0.5em 2em;
 }
+
+.button-right {
```
Comment (Sinetheta) [inline app/assets/stylesheets/components/_buttons.scss]: Bootstrap comes with helpers like this `.right`

File: app/assets/stylesheets/layouts/_header.scss [hunk 3]
```diff
@@ -1,4 +1,10 @@
 .header {
   background-color: $color-body;
   text-align: center;
+  margin-top: 40px;
+}
+
+.keg-picture {
+  width: 25%;
```
Comment (Sinetheta) [inline app/assets/stylesheets/layouts/_header.scss:8]: You might want to use their grid now for layout http://getbootstrap.com/css/#grid

Leaving this is fine too.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (DariaSova) [inline app/assets/stylesheets/application.scss:2]: I'm going to add it in the next PR (it doesn't change much)

File: app/assets/stylesheets/layouts/_header.scss (hunk 3 above)
Comment (DariaSova) [inline app/assets/stylesheets/layouts/_header.scss:8]: i decided to leave it like this for now

**Comment by Sinetheta** [comment]
:+1:

**Pull request merged**